.travis.yml

```diff
@@ -5,6 +5,10 @@ os:
   - linux
   - osx
 
+arch:
+  - amd64
+  - arm64
```

Comment:
This file has a line that says `matrix:` in it, under it add this:

```
  exclude:
    - os: osx
      arch: arm64
```

note that the first line starts with two spaces.

Reply:
modified.